userpatches/customize-image.sh

```diff
@@ -106,3 +106,6 @@ apt-cache policy yunohost | grep -Po 'Installed: \K.+' > /tmp/overlay/yunohost_v
 
 # Clean stuff
 apt clean
+
+# We don't want the damn network-manager :/
+apt-get purge -y -q network-manager 
```

Comment:
Is this still mandatory? cf. https://github.com/YunoHost/arm-images/blob/6fd3d260ccb053b43c6a27b489ea3bf0ef0fa0bd/userpatches/config-yunohost.conf#L24